shell/packages/sandstorm-db/migrations.js

```diff
@@ -238,10 +238,28 @@ function splitUserIdsIntoAccountIdsAndIdentityIds() {
   // form of API token.
 }
 
-function appUpdateSettings() {
-  Settings.insert({_id: "appMarketUrl", value: "https://apps.sandstorm.io"});
-  Settings.insert({_id: "appIndexUrl", value: "https://app-index.sandstorm.io"});
-  Settings.insert({_id: "appUpdatesEnabled", value: true});
+function appUpdateSettings() {}
+// This migration was written to populate initial values for the "appMarketUrl", "appIndexUrl",
+// and "appUpdatesEnabled" settings. We've replaced it with a no-op becuase it's more convenient
+// to lazily initialize these settings, returning the canonical values as defaults.
```

Comment:
There's a problem with this: If the setting is not present in the database *on the client side*, it's possibly because it isn't set (and we should use the default) but it's also possibly because the subscription isn't ready (in which case using the default is dangerous, as some paranoid users may explicitly not want to connect to the app market under any circumstances).

Is there actually a good reason for this change or is it just aesthetics?